mslib/msui/linearview.py

```diff
@@ -26,15 +26,17 @@
 """
 
 from mslib.utils.config import config_loader
-from PyQt5 import QtGui, QtWidgets
+from PyQt5 import QtGui, QtWidgets, QtCore
 from mslib.msui.qt5 import ui_linearview_window as ui
 from mslib.msui.qt5 import ui_linearview_options as ui_opt
 from mslib.msui.viewwindows import MSUIMplViewWindow
 from mslib.msui import wms_control as wms
 from mslib.msui.icons import icons
+from mslib.msui import autoplot_dockwidget as dock
 
```

Comment:
ToDo: similiar to the other names, not dock but may be apd, because  ad is occupied

Reply:
https://github.com/Open-MSS/MSS/pull/2548